tpp-app/src/onboarding/Welcome.js

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+import {StyleSheet, Text, Image, ImageBackground, TouchableOpacity} from 'react-native';
+import MNationIcon from '../../ios/tppapp/Images.xcassets/AppIcon.appiconset/1024.png'
+import OnboardingBackground from '../../ios/tppapp/Images.xcassets/SplashScreenBackground.imageset/background.png'
+import { NavigationContainer } from '@react-navigation/native';
+import { createNativeStackNavigator } from '@react-navigation/native-stack';
+import PeriodStart from './PeriodStart';
+import PeriodLength from './PeriodLength';
+import SymptomsChoices from './SymptomsChoices';
+import { MyTabs } from '../../App';
+
+const CustomStartButton = ({ onPress, title }) => (
+  <TouchableOpacity onPress={onPress} style={styles.appButtonContainer}>
+    <Text style={styles.appButtonText}>{title}</Text>
+  </TouchableOpacity>
+);
+
+// Get Start Page Component 
+const GetStarted = ({ navigation }) => (
+  <ImageBackground source={OnboardingBackground} style={styles.container}>
+    <Image style={styles.appIcon} source={MNationIcon}/>
+    <Text style={styles.titleText}>Welcome!</Text>
+    <CustomStartButton title="Get Started" onPress={() => navigation.navigate("Period Length")}/>
+  </ImageBackground>
+);
+
+// Stack Navigation for the Onboarding Pages 
+const Stack = createNativeStackNavigator();
+export default function Welcome() {
+  return (
+    <NavigationContainer>
+      <Stack.Navigator 
+        screenOptions={{
+          headerShown: false
+        }}
+      >
+      <Stack.Screen
+        name="Get Started"
+        component={GetStarted}
+      />
+      <Stack.Screen
+        name="Period Length"
+        component={PeriodLength}
+      />
+      <Stack.Screen
+        name="Period Start"
+        component={PeriodStart}
+      />
+      <Stack.Screen
+        name="Symptoms Choices"
+        component={SymptomsChoices}
+      />
+      <Stack.Screen 
+        name="Main Page"
+        component={MyTabs}
+      />
```

Comment:
Maybe you can add a dictionary that has all the screen names to prevent spelling mistakes when navigating to it in the future. (e.g. STACK_SCREENS = { "get_started" : "get_started", ...} )

Reply:
Love the suggestion - thanks Roy!